tests/cli/test_build.py

```diff
@@ -28,7 +29,7 @@ def mock_project_directory(
         root.mkdir()
         program.root = root
         program.files.cmake_build_dir = root / BUILD_DIR / build_subdir
-        program.files.cmake_config_file = root / CMAKE_CONFIG_FILE_PATH
+        program.files.cmake_config_file = root / CMAKE_CONFIG_FILE
```

Comment:
What would the full path to this file be?

Reply:
Mbed config cmake file was created at the root directory. I've moved it to build directory now.